### PR TITLE
[10.x] Ensure captured time is in configured timezone

### PR DESCRIPTION
**Pull request by timacdonald**
This is a bugfix to ensure that the captured "startedAt" time for requests / commands is  converted to use the applications timezone before being passed to the `handleRequestLifecycleLongerThan` and `handleCommandLifecycleLongerThan` functions.

### CODE REVIEW

File: src/Illuminate/Foundation/Http/Kernel.php [hunk 1]
```diff
@@ -214,6 +214,8 @@ public function terminate($request, $response)
 
         $this->app->terminate();
 
+        $this->requestStartedAt->setTimezone($this->app['config']->get('app.timezone') ?? 'UTC');
```
Comment (alirezasalehizadeh) [inline src/Illuminate/Foundation/Http/Kernel.php:217]: @timacdonald 
this
 `get('app.timezone') ?? 'UTC')`
 Shouldn't it be like this?
`get('app.timezone', 'UTC')`

Comment (alirezasalehizadeh) [reply src/Illuminate/Foundation/Http/Kernel.php:217]: @antonkomarev 
So, this means 
```php
$this->commandStartedAt->setTimezone($this->app['config']->get('app.timezone', 'UTC'));
``` 
should be changed to 
```php
$this->commandStartedAt->setTimezone($this->app['config']->get('app.timezone') ?? 'UTC');
``` 
in `console` kernel?

Comment (ianfortier) [reply src/Illuminate/Foundation/Http/Kernel.php:217]: I think this PR created an issue with Vapor :/
https://flareapp.io/share/yPa2l9K7

https://github.com/laravel/framework/issues/47591

Comment (timacdonald) [reply src/Illuminate/Foundation/Http/Kernel.php:217]: Fix: https://github.com/laravel/framework/pull/47596

**Pull request merged**